OracleDatabase/SingleInstance/dockerfiles/21.3.0/Dockerfile.xe

```diff
@@ -79,12 +81,15 @@ RUN cd $INSTALL_DIR && \
     rm -rf $INSTALL_DIR && \
     $ORACLE_BASE/oraInventory/orainstRoot.sh && \
     $ORACLE_HOME/root.sh && \
-    echo 'export ORACLE_SID=XE' > .bashrc
+    echo 'export ORACLE_SID=XE' >> /home/oracle/.bashrc && \
+    chown oracle.oinstall /home/oracle/.bashrc
 
 USER oracle
 WORKDIR /home/oracle
 
 HEALTHCHECK --interval=1m --start-period=5m --timeout=30s \
    CMD "$ORACLE_BASE/$CHECK_DB_FILE" >/dev/null || exit 1
 
-CMD exec $ORACLE_BASE/$RUN_FILE
+EXPOSE 1521/tcp 5500/tcp
```

Comment:
Why are we adding `EXPOSE` back in? We removed this years ago for good reasons.

Reply:
Hi @gvenzl 
Removed it. Added for easy of use / convenience. Didn't knew that it was removed earlier. Do you know the reason why it was removed? If yes, can you give the reason in brief.

Reply:
`EXPOSE` and `VOLUME` cannot be undone in downstream images. So if anybody uses this image as a base image, the ports stay exposed (or volume if `VOLUME` was used), which may not always be desired. On the flip side, neither `EXPOSE` nor `VOLUME` are needed to expose any port or location inside the container. Users can expose any ports via the `-p` option.